applications/search/src/pages/search-page/search-page.jsx

```diff
@@ -243,6 +243,19 @@ export const SearchPage = props => {
             />
           </Route>
         </Switch>
+        <div className="twitter-container">
+          <div className="twitter">
+            <h3>Nytt fra Datakatalogen på Twitter</h3>
+            <a
+              className="twitter-timeline"
+              data-width="600"
+              data-height="400"
+              href="https://twitter.com/datakatalogen?ref_src=twsrc%5Etfw"
+            >
+              Tweets by datakatalogen
```

Comment:
Samme for denne hvis den skal ha oversetting for tre språk?

Reply:
Den vises ikke på siden. Twitter gjør oversettingen